src/App.jsx

```diff
@@ -13,15 +13,115 @@ function App() {
     const [todoList, setTodoList] = useState([]);
     const [isLoading, setIsLoading] = useState(true);
 
-    useEffect(() => {
-        new Promise((resolve, reject) => {
-            setTimeout(() => {
-                resolve({ data: { todoList: [] } });
-            }, 2000);
-        }).then((result) => {
-            setTodoList(result.data.todoList);
+    // Get tasks from AirTable
+    const fetchData = async () => {
+        const apiURL = `https://api.airtable.com/v0/${
+            import.meta.env.VITE_AIRTABLE_BASE_ID
+        }/${import.meta.env.VITE_TABLE_NAME}`;
+        const options = {
+            method: "GET",
+            headers: {
+                Authorization: `Bearer ${
+                    import.meta.env.VITE_AIRTABLE_API_TOKEN
+                }`,
+            },
+        };
+        try {
+            const response = await fetch(apiURL, options);
+
+            if (!response.ok) {
+                const message = `Error has ocurred: ${response.status}`;
+                throw new Error(message);
+            }
+            const data = await response.json();
+            const todos = data.records.map((todo) => {
+                const newTodo = {
+                    id: todo.id,
+                    title: todo.fields.title,
+                };
+                return newTodo;
+            });
+            setTodoList(todos);
             setIsLoading(false);
-        });
+        } catch (error) {
+            console.log(error.message);
```

Comment:
If we are printing an error to the console, you can use `console.error(error)` instead and it will make it red and scary, which clearly indicates it is an error. We can do this for all three functions that call to Airtable.

Reply:
Is `console.error(error)` log out entire error object to the console? Is this way better for debugging?  If not, is it better to keep it this way since we only need basic error about the error?

Reply:
`console.error` will indeed print out the same information as console.log, which is the message and the call stack that led you to that error. It has other benefits though:
- Makes it red and noteworthy, highlighting it from the rest of your application's debugging output.
- Adds to the error counter in the top right of your dev tools (the red exclamation) to help indicate there was a problem even if you are in a different part of your dev tools.

When you see the error in your console, then you should debug why the error occurred by looking at the code, checking why your network request failed in the `network` tab of your dev tools, and whatever else helps you track down the error.